### ISSUE TITLE
Files with the same name and type obscure each other

### ISSUE THREAD

**Issue by MD485** [Facepunch/sbox-issues#4499]
### Describe the bug

When you have two files of the same name and asset type only the first instance of the file shows up in the asset view.

### To Reproduce

1. Pick a folder
2. Make a file called "test.png"
3. Make a file called "test.jpg" or "test.tga"
4. View folder in assets browser.

### Expected behavior

All files named test with a valid extension should show up in the folder.

### Media/Files

![image](https://github.com/Facepunch/sbox-issues/assets/36082767/4cc3f278-f515-42f2-a712-6623cffffc4a)
![image](https://github.com/Facepunch/sbox-issues/assets/36082767/87d14ed7-f877-462c-b5c6-bdaad94cab95)
![image](https://github.com/Facepunch/sbox-issues/assets/36082767/f4a2aada-e6af-4234-8d28-8814eacb1eca)
![image](https://github.com/Facepunch/sbox-issues/assets/36082767/2be6d843-5fb1-4787-95c6-c463272aac00)

### Additional context

I imagine this only really applies to Images and Audio? I think all the other asset types would have conflicts if two files had the same name in the same folder.

Maybe this is happening because of code written to filter out vdml_c files?

**Comment by handsomematt** [comment]
png and tga are the same image type, this wouldn't happen with test.vmdl and test.png afaik

**Comment by MD485** [comment]
This should only happen for files of the same type yeah, I think only the 'Image' and 'Sound File' types should have this issue, they're the only ones I can think of that support multiple extensions.

**Comment by garrynewman** [comment]
Yeah this kind of sucks, but it isn't something we can fix any time soon
